shap/_explanation.py

```diff
@@ -866,12 +872,12 @@ def _auto_cohorts(shap_values, max_cohorts):
                         name += " >= "
                     name += str(threshold) + " & "
         path_names.append(name[:-3])  # the -3 strips off the last unneeded ' & '
-    path_names = np.array(path_names)
+    path_names_arr = np.array(path_names)
```

Comment:
There have been a few cases like this where mypy complains if a variable starts life as e.g. a list and then is converted to an array. A simple fix is just to give one of the variables a different name.

Reply:
yes, I think this is actually a useful feature of mypy since it is the cause of a lot of bugs I've seen/produced